<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TermsConditions</class>
 <widget class="QWidget" name="TermsConditions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10,0">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Terms And Conditions</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="titleLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="delivery_TimeLabel">
       <property name="text">
        <string>Delivery_Time</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="paymentTermsLabel">
       <property name="text">
        <string>Payment Terms</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="warrantyLabel">
       <property name="text">
        <string>Warranty</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="validityLabel">
       <property name="text">
        <string>Validity</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="exclusionsLabel">
       <property name="text">
        <string>Exclusions</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="extraInfoLabel">
       <property name="text">
        <string>Extra Info</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPlainTextEdit" name="DeliveryplainTextEdit"/>
     </item>
     <item row="2" column="1">
      <widget class="QPlainTextEdit" name="paymentplainTextEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="QPlainTextEdit" name="WarrantyplainTextEdit"/>
     </item>
     <item row="4" column="1">
      <widget class="QPlainTextEdit" name="ValidityplainTextEdit"/>
     </item>
     <item row="5" column="1">
      <widget class="QPlainTextEdit" name="ExclusionsplainTextEdit"/>
     </item>
     <item row="6" column="1">
      <widget class="QPlainTextEdit" name="ExtraTextEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
